terraform-eks/1-variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "initials" {
+  description = "Enter your initials to include in URLs. Lowercase only!!!"
+  default     = "tm"
```

Comment:
Can we remove the initials/region/AWS id/aws role. These can be set locally in a tfvar files.